docs/examples.rst

```diff
@@ -18,3 +18,4 @@ General-purpose and introductory examples from PlasmaPy.
     :reversed:
 
     notebooks/*
+    notebooks/*/*
```

Comment:
@StanczakDominik I think we're starting to get enough examples that we should start breaking these up into sub-sections of notebooks.  What do you think?  (Obviously we'd do this in a separate PR.)

Reply:
As in, in the main notebook list? Yeah, definitely; definitely calling that out of pr scope as well though.